apps/balancer-tools/src/app/apr/(components)/HeaderEndButton.tsx

```diff
@@ -39,21 +44,24 @@ export default function HeaderEndButton() {
   }) => {
     router.push(`/apr/pool/${networkFor(network)}/${poolId}`);
   };
-
+  const avaliableNetworks = Object.keys(networksOnBalancer).map((key) => ({
+    value: key,
+    label: capitalize(networksOnBalancer[key]),
+  }));
```

Comment:
should probably be contingent on the available pools list... but that also creates a dependency on pools with gauges 😓